### PR TITLE
Spanish translation added :D

### PR DESCRIPTION
**Pull request by MrJako2001**

### CODE REVIEW

File: src/data/languages/Spanish.json [hunk 1]
```diff
@@ -0,0 +1,444 @@
+{
+	"info": {
+		"language": "Español",
+		"language_english": "Spanish",
+		"fallback": ["English"],
+		"maintainer": "MrJako2001",
+		"maintainer_link": "https://anilist.co/user/MrJaco/",
+		"discussion_link": "",
```
Comment (hohMiyazawa) [inline src/data/languages/Spanish.json:8]: Add https://github.com/hohMiyazawa/Automail/issues/69 as the discussion link if you don't want to manage your own discussion place.

File: src/data/languages/Spanish.json [hunk 2]
```diff
@@ -0,0 +1,444 @@
+{
+	"info": {
+		"language": "Español",
+		"language_english": "Spanish",
+		"fallback": ["English"],
+		"maintainer": "MrJako2001",
+		"maintainer_link": "https://anilist.co/user/MrJaco/",
+		"discussion_link": "",
+		"notes": "This is the default language of Automail. This key should have all the translation keys, while other translations don't have to. I'm not a native speaker, but I try to stick to British English here (feel free to britpick!). You can make translation files for other version of English too if you want, name it something like 'English (US)', and put 'English' in the fallback array. You need only include the keys that are different."
```
Comment (hohMiyazawa) [inline src/data/languages/Spanish.json:9]: That's the notes for the default English language file, leave blank "" if you have no notes specific to Spanish.

**Comment by hohMiyazawa** [comment]
Thanks!

File: src/data/legacyModuleDescriptions.json [hunk 3]
```diff
@@ -264,6 +264,6 @@
 	"extendedDescription": "Partial localisation. Does not translate the whole site.\nFor Japanese, you may also want to change your title language at https://anilist.co/settings/media\n\nWant to contribute? See: https://github.com/hohMiyazawa/Automail/issues/69",
 	"type": "select",
 	"categories": ["Script","Newly Added"],
-	"values": ["English","Norsk","Português","Deutsch","日本語","Åarjelsaemie","Svenska","English (US)"]
+	"values": ["English","Norsk","Português","Deutsch","日本語","Åarjelsaemie","Svenska","English (US)","Spanish"]
```
Comment (hohMiyazawa) [inline src/data/legacyModuleDescriptions.json:267]: Use "Español" here, and move it between "Norsk" and "Português" to keep it sorted by completeness. (To make it more complete than Norwegian too, you would have to add staff credit translations, see "special_keys.txt" and the bottom half of "Norwegian.json" for that)

File: src/localisation.js [hunk 4]
```diff
@@ -8,6 +8,7 @@ const languageFiles = {
 	"Norsk": m4_include(data/languages/Norwegian.json),
 	"Svenska": m4_include(data/languages/Swedish.json),
 	"English (US)": m4_include(data/languages/English_US.json)
+	"Spanish": m4_include(data/lenguajes/Spanish.json)
```
Comment (hohMiyazawa) [inline src/localisation.js:11]: Wrong path, all translations go in the "languages" directory.

File: src/localisation.js [hunk 5]
```diff
@@ -8,6 +8,7 @@ const languageFiles = {
 	"Norsk": m4_include(data/languages/Norwegian.json),
 	"Svenska": m4_include(data/languages/Swedish.json),
 	"English (US)": m4_include(data/languages/English_US.json)
```
Comment (hohMiyazawa) [inline src/localisation.js:10]: comma on the end of this line

File: src/data/legacyModuleDescriptions.json (hunk 3 above)
Comment (hohMiyazawa) [reply src/data/legacyModuleDescriptions.json:267]: To be clear, credit translations are completely optional, and you don't have to do that for this pull request to be merged.

File: src/data/languages/Spanish.json [hunk 6]
```diff
@@ -0,0 +1,444 @@
+{
+	"info": {
+		"language": "Español",
+		"language_english": "Spanish",
+		"fallback": ["English"],
```
Comment (hohMiyazawa) [inline src/data/languages/Spanish.json:5]: Question: Would it make sense to have Portuguese as a fallback before English? That is, if there's a Spanish key missing, the script will first see if there's a Portuguese one before using the English key.

Comment (hohMiyazawa) [reply src/data/languages/Spanish.json:5]: The syntax would be ``"fallback": ["Portuguese", "English"],``

File: src/data/languages/Spanish.json [hunk 7]
```diff
@@ -0,0 +1,444 @@
...
+"$time_1year": "Hace un año",
+"$time_Myear": "Hace {0} años",
+"$time_medium_Mday": "días",
+"$time_short_second": "s",
+"$time_short_minute": "m",
+"$time_short_hour": "h",
+"$time_short_day": "d",
+"$time_short_week": "w",
+"$time_short_month": "m",
+"$time_short_year": "y",
```
Comment (hohMiyazawa) [inline src/data/languages/Spanish.json:37]: This appears to be the English abbreviations. Is that intentional?

File: src/data/languages/Spanish.json [hunk 8]
```diff
@@ -0,0 +1,444 @@
...
+"$notification_forumMention": " te mencionó, en el hilo del foro ",
+"$notifications_softBlock": "Bloquéo suave de usuarios",
+"$notifications_hideLike": "Ocultar notificaciones de \"Me gusta\"",
+"$notifications_showHoh": "Mostrar notificaciones de hoh",
+"$notifications_showDefault": "Mostrar notificaciones nativas",
+"$notifications_comments": "comentarios",
+"$notifications_button_reset": "Marcar todo como leído",
+"$documentTitle_notifications": "Notificaciones · AniList",
+"$documentTitle_home": "Inicio · AniList",
+"$documentTitle_forum": "Foro -Discusión de Anime y Manga · AniList",
```
Comment (hohMiyazawa) [inline src/data/languages/Spanish.json:182]: Space after the dash

File: src/data/languages/Spanish.json (hunk 7 above)
Comment (MrJako2001) [reply src/data/languages/Spanish.json:37]: I thought it more appropriate to leave them as they are.

File: src/data/languages/Spanish.json (hunk 6 above)
Comment (MrJako2001) [reply src/data/languages/Spanish.json:5]: It is better in English as Portuguese is not widely spoken in Latin America.

File: src/data/legacyModuleDescriptions.json (hunk 3 above)
Comment (MrJako2001) [reply src/data/legacyModuleDescriptions.json]: Credit translations added for a better experience :D

File: src/localisation.js (hunk 5 above)
Comment (MrJako2001) [reply src/localisation.js]: Fixed

File: src/localisation.js (hunk 4 above)
Comment (MrJako2001) [reply src/localisation.js]: Fixed

File: src/data/legacyModuleDescriptions.json (hunk 3 above)
Comment (MrJako2001) [reply src/data/legacyModuleDescriptions.json]: Ordered

File: src/data/languages/Spanish.json (hunk 2 above)
Comment (MrJako2001) [reply src/data/languages/Spanish.json]: Fixed

File: src/data/languages/Spanish.json (hunk 1 above)
Comment (MrJako2001) [reply src/data/languages/Spanish.json]: Fixed

**Comment by hohMiyazawa** [comment]
Since it appears you haven't had the setup to compile automail, here's a proof build in case you want to test that it all looks right before it goes live:

https://share.wildbook.me/RscTVAKISub6qznE.js

**Comment by hohMiyazawa** [comment]
Live online as of v9.99.70

**Pull request merged**